core/processor/src/main/java/io/quarkus/annotation/processor/generate_doc/SummaryTableDocFormatter.java

```diff
@@ -89,7 +89,7 @@ public void format(Writer writer, ConfigDocKey configDocKey) throws IOException
     public void format(Writer writer, ConfigDocSection configDocSection) throws IOException {
         String anchor = anchorPrefix + getAnchor(configDocSection.getSectionDetailsTitle());
```

Comment:
Hi @dufoli thanks for trying the fix out,

I think that it is not working because the changes from the fix were not picked.

This should be:
```
String anchor = anchorPrefix + getAnchor(configDocSection.getName());
 final String sectionRow = String.format(TABLE_SECTION_ROW_FORMAT, anchor, anchor,
 configDocSection.getSectionDetailsTitle());
 writer.append(sectionRow)
```

Can you try that out locally with `mvn clean install -DskipTests -DskipITs` and see if it fixes the issue? Thanks.